docs/contributor/image_mode.md

```diff
@@ -0,0 +1,217 @@
+# Image Mode for MicroShift
+
+Image mode is a new approach to operating system deployment that lets users build,
+deploy, and manage Red Hat Enterprise Linux as a bootable container (`bootc`) image.
+MicroShift build and deployment procedures can utilize bootable containers to
+benefit from this technology.
+
+This document demonstrates how to:
+* Create a bootable container image and boot a RHEL operating system from this
+image using `podman`
```

Comment:
It is not booting the RHEL operating system using `podman run` command but starting the container.